lib/ohai/plugins/linux/platform.rb

```diff
@@ -266,7 +266,8 @@ def determine_platform_family
     elsif File.exist?("/etc/alpine-release")
       platform "alpine"
       platform_version File.read("/etc/alpine-release").strip
-    elsif File.exist?("/usr/lib/os-release")
+    # If /etc/os-release file exists, we take that as source of truth
+    elsif File.exist?("/usr/lib/os-release") && !File.exist?("/etc/os-release")
```

Comment:
@tas50 Does this make sense? ` && !File.exist?("/etc/os-release")` essentially ensures openSUSE Leap and SLES not match with this check.

Reply:
CI failures doesn't seem related to my change. :thinking: